.github/workflows/deploy.yml

```diff
@@ -24,7 +24,11 @@ jobs:
         touch $HOME/.gem/credentials
         chmod 0600 $HOME/.gem/credentials
         printf -- "---\n:rubygems_api_key: ${RUBYGEMS_API_KEY}\n" > $HOME/.gem/credentials
+        printf -- ${PRIVATE_KEY_PEM} > ${SIGNING_KEY}
```

Comment:
Silly question, but is this actually printing the private key to the logs?

Reply:
No it shouldn't as GH Actions should detect it matches one of the secrets. It doesn't print the api key either if you check past deploy logs.

Reply:
Cool, thx!